### PR TITLE
fix: frappe listing template fixed

### PR DESCRIPTION
**Pull request by nehasacher143**
**Task:** Fix the listing template, so that, even if we have nothing to show.

### CODE REVIEW

**Review by DeveloperMetal — COMMENTED** [review]
@nehasacher143 What's this PR about?
Are you making the header appear when there are no results?

What are the side effects on the rest of the system?

**Comment by nehasacher143** [comment]
Hey @DeveloperMetal 
Yes, I am trying to make the header appear, when there are no results.
I guess this won't have any adverse effects on the rest of the site, as it will only work if we have this in Context, and for now we are only using it for our new website design

File: frappe/templates/includes/list/list.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {% if sub_title %}
 <h4 class="text-muted">{{ sub_title }}</h4>
 {% endif %}
+
+{% if result_heading_template %}{% include result_heading_template %}{% endif %}
```
Comment (hrwX) [inline frappe/templates/includes/list/list.html]: - Do not use `result heading template` variable here, as the name suggests that the template is for result
- You can add a new variable `page heading template` to achieve this

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by nehasacher143** [comment]
Hey @hrwX Please review this

**Review by hrwX — APPROVED** [review]

**Pull request merged**